src/Illuminate/Database/Migrations/Migration.php

```diff
@@ -2,8 +2,17 @@
 
 namespace Illuminate\Database\Migrations;
 
+use Illuminate\Support\Facades\Schema;
+
 abstract class Migration
 {
+    /**
+     * The database schema.
+     *
+     * @var \Illuminate\Support\Facades\Schema
```

Comment:
@antonkomarev Shouldn't this be `\Illuminate\Database\Schema\Builder`?

Reply:
@X-Coder264 I think you right, I've thought about it too.